### ISSUE TITLE
aria-busy does not work in live region

### ISSUE THREAD

**Issue by JAWS-test** [FreedomScientific/VFO-standards-support#257]
### Summary

aria-busy does not work in live region according to https://rawgit.com/w3c/aria/master/#aria-busy

1. Save as HTML file:
```
<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="utf-8">
		<title>aria-busy</title>
	</head>
	<body>
		<div aria-live="polite" aria-busy="true" id="1">busy</div>
		<button onClick="document.getElementById('1').textContent='not busy';">Change live region</button>
		<button onClick="document.getElementById('1').setAttribute('aria-busy','false');">Change aria-busy</button>
	</body>
</html>
```
2. Press button "Change live region" (changes the text content inside live region)
3. Press button "Change aria-busy" (changes form true to false)

### Expected result

* Press button "Change live region": no output
* Press button "Change aria-busy": output of the new text content "not busy"

### Actual result

* Firefox and Chrome: Output is done despite aria-busy=true
* IE 11: No output after changing from aria-busy=true to false

#### JAWS version and build number

JAWS 2019.1907.42
 
#### Operating System and version

Windows 8

#### Browser and version:

Internet Explorer 11.0.9600.19431
Chrome 76.0.3809.100
Firefox 68.0.2

**Comment by JAWS-test** [comment]
See also https://github.com/w3c/aria-practices/issues/78 and many other problems with live regions: https://github.com/FreedomScientific/VFO-standards-support/issues?q=live+region

**Comment by stevefaulkner** [comment]
[test case](https://codepen.io/stevef/pen/mdowVyW) tested in JAWS 2024 AND LATEST chrome/firefox does not work as expected, works as expected with NVDA

**Comment by BrettLewisVispero** [comment]
Imported into ADO as bug 68617

**Comment by JeremiahRogersVispero** [comment]
@JAWS-test, this issue appears to be fixed in JAWS 2026.2606.132. The code I used to test is below.

`<!DOCTYPE html>`
`<html lang="en-us">`
`	<head>`
`		<title>Code Sample for Issue 257 from the Vispero Standards Support GitHub Repository</title>`
`	</head>`
`	<body>`
`		<p>This intro paragraph is here to have text and a tab stop before any material being tested. <a href="https://www.vispero.com/">Vispero Home Page</a></p>`
`		<main aria-labelledby="lblMain">`
`			<h1 id="lblMain">Code Sample for <a href="https://github.com/FreedomScientific/standards-support/issues/257">Issue 257</a></h1>`
`			<div aria-live="polite" aria-busy="false" id="1">This content will be replaced with numbers when Change live region is clicked.</div>`
`			<button onClick="staggerContent();">Change live region</button>`
`			<div id="2" aria-live="polite">Aria-busy was set to false on page load</div>`
`		</main>`
`		<p>This exit paragraph is here to have text and a tab stop after any material being tested. <a href="https://vispero.com/resources/category/blog/">Vispero Blog</a></p>`
`	</body>`
`	<script>`
`		function setContent (msgContent) {`
`			document.getElementById('1').textContent=msgContent;`
`		}`
`		function sleep(ms) {`
`			return new Promise(resolve => setTimeout(resolve, ms));`
`		}`
`		async function staggerContent() {`
`			staggerTime=2000`
`			document.getElementById('1').setAttribute('aria-busy','true');`
`			document.getElementById('2').textContent="Aria-busy is set to "+document.getElementById('1').getAttribute('aria-busy');`
`			setContent("1");`
`			await sleep(staggerTime);  // Pause for 2000 milliseconds, or 2 seconds`
`			setContent("2");`
`			await sleep(staggerTime);  // Pause for 2000 milliseconds, or 2 seconds`
`			setContent("3");`
`			document.getElementById('1').setAttribute('aria-busy','false');`
`			document.getElementById('2').textContent="Aria-busy is set to "+document.getElementById('1').getAttribute('aria-busy');`
`		}`
`	</script>`
`</html>`
